docs/content/en/docs/pipeline-stages/builders/buildpacks.md

```diff
@@ -25,9 +25,17 @@ The following options can optionally be configured:
 
 {{< schema root="BuildpackArtifact" >}}
 
+**Builder**
+
 `builder` is *required* and tells Skaffold which
 [Builder](https://buildpacks.io/docs/app-developer-guide/build-an-app/) to use.
 
+**User defined environment variables**
+
+`env` makes it possible to pass arbitrary environment variables to the buildpacks.
```

Comment:
_pass_ makes it sounds like we expose variables in the user's current environment. This is more constrained, right?
```suggestion
`env` makes it possible to configure specific environment variables for buildpacks.
Many buildpacks use environment variables to adjust their detection and the build phases,
such as selecting specific versions of language runtimes.
Note that user's current environment is not passed through to buildpacks.
```

Reply:
I'll fo that